Projects/19-Captsone_01/README.md

```diff
@@ -0,0 +1,830 @@
+# Insights into the I-SPY clinical trial
```

Comment:
No, I created a separate repository for the project and simply activated github pages there. 
some info here:

https://help.github.com/articles/user-organization-and-project-pages/

all you need is a README.md file and go to the settings section to activate github pages